database/seeders/UserSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class UserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        DB::table('users')->insert([
+            'name' => 'Aman',
+            'email' => 'amanbahuguna009@gmail.com',
```

Comment:
Can we write like this?
```suggestion
            'name' => 'poolYourRide Admin',
            'email' => 'pollyourride@gmail.com',
```

Reply:
It's just a seeder. Credentials can have any value